include/surface_perception/segmentation.h

```diff
@@ -157,14 +163,15 @@ bool GetSceneAboveSurface(pcl::PointCloud<pcl::PointXYZRGB>::Ptr cloud,
                           pcl::PointIndicesPtr indices,
                           const pcl::ModelCoefficients& coefficients,
                           double margin_above_surface,
+                          const float& height_limit,
                           pcl::PointIndices::Ptr above_surface_indices);
 
 /// \brief The algorithm that segments objects above a given surface.
```

Comment:
Update the documentation: it "segments the objects above each of a list of surfaces."

Reply:
Completed